### PR TITLE
reenable `spectate`

### PR DESCRIPTION
**Pull request by ab9rf**
by request from the community

do _not_ include in 50.10-r1, this should go through a beta cycle before being added to production

### CODE REVIEW

File: docs/changelog.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@ Template for new versions:
 ## New Tools
 
 ## New Features
+- `spectate`: automatically follow productive dwarves (returned to availability)
```
Comment (myk002) [inline docs/changelog.txt:57]: this would be a "New Tool", ne?

**Review by myk002 — COMMENTED** [review]
also the docs probably need to have the "unavailable" tag removed

File: docs/changelog.txt (hunk 1 above)
Comment (ab9rf) [reply docs/changelog.txt:57]: lol yes bad me

**Comment by ab9rf** [comment]
closing because the merge history is ugly, will reopen

**Pull request closed without merging**